lib/SILOptimizer/Mandatory/MandatoryCombine.cpp

```diff
@@ -156,6 +214,12 @@ void MandatoryCombiner::addReachableCodeToWorklist(SILFunction &function) {
       ++iterator;
 
       if (isInstructionTriviallyDead(instruction)) {
```

Comment:
We're already "doing the work" in Onone. We should either just erase the known-dead instruction, or not call `isInstructionTriviallyDead` at all (preferably the later).

Reply:
We need to know to skip it. We are keeping the size of the worklist down.

Reply:
Fair enough.